helper_scripts/create_gpg_keys.py

```diff
@@ -65,8 +65,8 @@ def __call__(self, parser, namespace, values, option_string=None):
     log.info("Writing to %s.{pub,sec}", name)
     for pvt in (True, False):
         key = scriptworker.gpg.export_key(gpg, fingerprint, private=pvt)
-        suffix = ".pub"
+        filename = "{}.pub".format(host)
```

Comment:
IIUC, the file name is now `fqdn.pub`. However you said in https://github.com/mozilla-releng/scriptworker/pull/202#issue-180680915, you compromised to `user@fqdn.pub`. Which one is expected?